test/tests/functional/pbs_sched_attr_updates.py

```diff
@@ -132,8 +134,8 @@ def test_accrue_type(self):
         # sched will send all updates for the jobs
         jid4 = self.server.submit(Job())
         self.scheduler.run_scheduling_cycle()
-        self.server.expect(JOB, "comment", op=SET, id=jid4, max_attempts=5)
-        self.server.expect(JOB, {"accrue_type": "1"}, id=jid4, max_attempts=5)
-        self.server.expect(JOB, "comment", op=SET, id=jid3, max_attempts=5)
-        self.server.expect(JOB, {"accrue_type": "1"}, id=jid3, max_attempts=5)
-        self.server.expect(JOB, {"accrue_type": "1"}, id=jid2, max_attempts=5)
+        self.server.expect(JOB, "comment", op=SET, id=jid4, max_attempts=1)
+        self.server.expect(JOB, {"accrue_type": "1"}, id=jid4, max_attempts=1)
+        self.server.expect(JOB, "comment", op=SET, id=jid3, max_attempts=1)
+        self.server.expect(JOB, {"accrue_type": "1"}, id=jid3, max_attempts=1)
+        self.server.expect(JOB, {"accrue_type": "1"}, id=jid2, max_attempts=1)
```

Comment:
If you are going to do max_attempts=1, then do a status(), and use asserts.  This means you'll do one qstat -f instead of 5.

Reply:
status returns a dict, not a list, so I'll have to iterate over it, match the jid and then do asserts accordingly. I kind of feel like 5 qstats is not a big deal, we often do max_attempts=5 anyways for existence=False. So I'd like to keep it as is, this will make the test more readable, easier to modify in future. Let me know what you think.